crates/autopilot/src/solvable_orders.rs

```diff
@@ -78,6 +77,7 @@ pub struct SolvableOrdersCache {
     ethflow_contract_address: Option<H160>,
     weth: H160,
     limit_order_price_factor: BigDecimal,
+    fee_policies: domain::ProtocolFee,
```

Comment:
nit: only holds a single policy at the moment. Same applies to other variable names in this file.
```suggestion
    fee_policy: domain::ProtocolFee,
```

Reply:
fixed